pkgs/development/libraries/gstreamer/good/default.nix

```diff
@@ -60,6 +61,16 @@ stdenv.mkDerivation rec {
     hash = "sha256-tnsxMTpUxpKbgpadQdPP3y9Y21c/tfSR5rul2ErqB3g=";
   };
 
+  patches = lib.optionals qt5Support [
```

Comment:
Ideally, I think this kind of patches should be applied no matter what, to make sure they don't bit rot. However, since this change is targeting `staging-next`, perhaps it _is_ the right thing to do, though I'm not sure. If that's indeed the case, a comment should explain the situation, and hopefully on the next update people will remember to check whether the patch is still required, and if yes whether it still applies.

Reply:
Yep, it was conditional to spare rebuilds in the current staging-next cycle. I would have made a separate PR to staging to undo the conditional patch, but I don't imagine this will need to be here long and so didn't really bother

I'll update the comment to make the situation clearer, though, because it could definitely be clearer on why it is conditional and when it can be removed

I already have in my notes to test the various flags during version bumps, so we'll know pretty quick when it's no longer needed since the build will fail due to being unable to apply the patch (most notably of nheko, which requires `qt5Support` on gst-plugins-good)

Reply:
I think it's more important to add the Qt variant to standard test set when you touch this package.